source/_posts/2019-01-24-nest-cannot-access-data.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Nest locks users out of their own data"
+description: "Nest is no longer accepting new developer accounts or certifications, blocking you from accessing your data."
+date: 2019-01-24 00:01:00
+date_formatted: "January 24, 2019"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories: Public-Service-Announcement
+---
+
+Some sad news for users with a Nest product: Nest is no longer accepting new developer accounts. This means that if you bought a Nest thermostat, you are no longer able access YOUR data.
+
+New Home Assistant users will not be able to integrate their Nest products. Home Assistant users that have already integrated Home Assistant with Nest are not affected.
+
+We hope that Nest will actually implement a new program and we will update this post when they do. Until then, we can no longer recommend buying or using any Nest product.
```

Comment:
Local control would be even better right? The number of IoT offers with local control is getting lower every day...

```suggestion
We hope that Nest will implement a new program or add support for locally controlling the Nest thermostat, if they do, we will update this post. Until then, we can no longer recommend buying or using any Nest product.
```